integration/error.go

```diff
@@ -0,0 +1,11 @@
+package integration
+
+import "github.com/giantswarm/microerror"
+
+var waitTimeoutError = microerror.New("waitTimeout")
+
+var tooManyResultsError = microerror.New("too many results")
+
+var unexpectedStatusPhaseError = microerror.New("unexpected status phase")
+
+var notFoundError = microerror.New("not found")
```

Comment:
Good that microkit is used here. We'd normally have error matchers too. Not sure if they are needed here.

Reply:
Good point added them just in case. IMO there's a lot of code in this test that could be moved to e2e-harness so that it can be reused in more project. Anyway, we can do that later.

I've also taken the opportunity to add the missing build tag to the errors file.